hack/update-vendor-licenses.sh

```diff
@@ -208,12 +204,16 @@ Options:
 __EOF__
       exit 9
   fi
-  cat "${file}"
 
-  echo
-  echo "= ${file} $(md5sum < "${file}" | awk '{print $1}')"
-  echo "================================================================================"
-  echo
-done >> ${TMP_LICENSE_FILE}
+  
+    cat "${file}"
+    echo
+    echo "= ${file} $(md5sum < "${file}" | awk '{print $1}')"
+  } >> "${TMP_LICENSE_FILE}"
+
+  license_file_name=$(echo "${PACKAGE}" | sed "s/\//_/g")
+  mv "${TMP_LICENSE_FILE}" "${TMP_VENDOR_LICENSE_DIR}/${license_file_name}"
+done
 
-cat ${TMP_LICENSE_FILE} > ${VENDOR_LICENSE_FILE}
+rm -rf "${VENDOR_LICENSE_DIR}"
```

Comment:
Because we changed `${VENDOR_LICENSE_DIR}` to be fully qualified `${KUBE_ROOT}/vendor/ALL_LICENSES` above, this can become:
```suggestion
if [[ -d "${VENDOR_LICENSE_DIR}" ]]; then
  rm -r "${VENDOR_LICENSE_DIR}"
fi
```

Reply:
@cblecker , Thank you for you review. I updated the code diff, thanks.